PythonRegression/tests/features/machine4/config.yml

```diff
@@ -0,0 +1,14 @@
+defaults: &db_1
+  db: https://s3.amazonaws.com/iota-db-files/dbs/machine4/m4_side_tangle_db.tar
```

Comment:
Just a question:
This s3 bucket is IF owned?

Reply:
It is, Giorgio manages it. It is the storage backend of https://dbfiles.iota.org/.

Reply:
Wrong actually, this one is stored on my own s3 bucket, but if there's an iota owned bucket I can start uploading these db's to, that would be preferred. Even if it's just uploading to the dbfiles.iota.org bucket.

Reply:
There is an IF bucket to use and Gal has the information required